web/client/plugins/Search.jsx

```diff
@@ -92,7 +103,13 @@ const ToggleButton = require('./searchbar/ToggleButton');
 /**
  * Search plugin. Provides search functionalities for the map.
  * Allows to display results and place them on the map. Supports nominatim and WFS as search protocols
+ * @prop {boolean} [showOptions=true] shows the burger menu in the input field
```

Comment:
are you sure these properties are for the Search plugin too?

Reply:
Yes @MV88, SearchBar component uses these props.

Reply:
ok, can you move these, together with the others definitions ?